textures.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+# filenames of all the PBR textures included in game in the textures folder, stored as a dict for each file
+texture_folder = "textures"
+
+normal_options = "normal_enabled = true\nnormal_scale = -1.0\n"
+depth_options = "depth_enabled = true\ndepth_scale = -0.01\ndepth_deep_parallax = false\ndepth_flip_tangent = false\ndepth_flip_binormal = false\n"
+
+class Texture:
+    def __init__(self, folder, jpg_dict, uv1_scale = ""):
+        self.folder = folder
+        self.jpg_dict = jpg_dict
+        # uv shouldn't be in the dict because it doesn't need an ext_resource like the .jpgs
+        self.uv1_scale = uv1_scale 
+
+
+ceiling_tile = Texture(
+    folder = "CeilingTile",
+    jpg_dict = {
+        "albedo" : "OfficeCeiling001_1K-JPG_Color.jpg",
+        "roughness" : "OfficeCeiling001_1K-JPG_Roughness.jpg",
+        "normal" : "OfficeCeiling001_1K-JPG_NormalGL.jpg", 
+        "depth" : "OfficeCeiling001_1K-JPG_Displacement.jpg"
+    },
```

Comment:
To reiterate use a json file here. Then make a classmethod to generate a new texture

``` python
@classmethod
def make_texture_from_json(json_obj): 
     return Texture(**json_obj)
```